README_BCtable.md

```diff
@@ -96,10 +104,16 @@ output stream to be used by the next filter. This is a list of available filters
 **convert2grb**
 :  Converts the stream to GRIB format
 
-**day_to_6h**
-:  Temporal interpolation in time from daily to 6-hourly frequency. Recommended
+**Mh_to_Nh**
```

Comment:
This filter name is quite cryptic. Maybe `time_interp_to_interval`?

Reply:
I agree. time_interp_to_interval looks much better.